src/Internal/DependencyManagement/CoreServiceProvider.php

```diff
@@ -232,6 +235,7 @@ function ( ...$arguments ) {
 		// Inbox Notes
 		$this->conditionally_share_with_tags( ContactInformationNote::class );
 		$this->conditionally_share_with_tags( CompleteSetupNote::class );
+		$this->conditionally_share_with_tags( ReviewAfterClicksNote::class, MerchantMetrics::class, WP::class );
```

Comment:
As @nima-karimi [astutely pointed out](https://github.com/woocommerce/google-listings-and-ads/pull/1016#discussion_r716578920) in another class, here the `Notes` classes and their subclasses don't actually implement `Conditional`, so there's no need to use `conditionally_share_with_tags`.

Reply:
Good point!